### ISSUE TITLE
AS 3.3 Build errors

### ISSUE THREAD

**Issue by bekuno** [cgeo/cgeo#7237]
##### Detailed steps causing the problem:
- Install/Upgrade Android Studio to version 3.3

##### Actual behavior after performing these steps:
Build works not more. Maybe from this error:
"D:\git\cgeo\main\build\intermediates\incremental\mergeBasicDebugResources\merged.dir\values\values.xml:3609: AAPT: error: resource android:attr/preserveIconSpacing is private.
error: failed linking references.
	at com.android.builder.internal.aapt.v2.Aapt2Exception$Companion.create(Aapt2Exception.kt:45)
...
Caused by: com.android.builder.internal.aapt.v2.Aapt2Exception: Android resource linking failed"

##### Expected behavior after performing these steps:
Gradle sync/Build with warnings as before

##### Version of c:geo used:
current master branch

##### Is the problem reproducible for you?
Yes

##### System information:
-

##### Other comments and remarks:
I think this point in GradleSync is the reason:
"WARNING: The following project options are deprecated and have been removed: 
android.enableAapt2
This property has no effect, AAPT2 is now always used.
Affected Modules: cgeo-contacts, main, mapswithme-api"

Maybe also this point:
"WARNING: Configuration 'compile' is obsolete and has been replaced with 'implementation' and 'api'.
It will be removed at the end of 2018. For more information see: http://d.android.com/r/tools/update-dependency-configurations.html
Affected Modules: main"

**Comment by moving-bits** [comment]
With studio 3.2.1 I get similar (but not identical) messages, without blocking the build process:

- WARNING: The option 'android.enableAapt2' is deprecated and should not be used anymore.
 Use 'android.enableAapt2=true' to remove this warning.
 It will be removed at the end of 2018..

- WARNING: Configuration 'compile' is obsolete and has been replaced with 'implementation' and 'api'.
 It will be removed at the end of 2018. For more information see: http://d.android.com/r/tools/update-dependency-configurations.html

**Comment by bekuno** [comment]
I have checked mit the new master branch. 
AS 3.3 is ok with the current gradle configuration.
It is also ok to use the gradle plugin 3.3 and gradle 4.10.2.
(in build.gradle:
        classpath 'com.android.tools.build:gradle:3.3.0'
und gragle-wrapper.properties:
distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.2-all.zip
)

After switch to the new configuration there is a new warning in addition to the already known:
_WARNING: API 'variantOutput.getInstall()' is obsolete and has been replaced with 'variantOutput.getInstallProvider()'.
It will be removed at the end of 2019.
For more information, see https://d.android.com/r/tools/task-configuration-avoidance.
To determine what is calling variantOutput.getInstall(), use -Pandroid.debug.obsoleteApi=true on the command line to display a stack trace.
Affected Modules: main

WARNING: Configuration 'compile' is obsolete and has been replaced with 'implementation' and 'api'.
It will be removed at the end of 2018. For more information see: http://d.android.com/r/tools/update-dependency-configurations.html
Affected Modules: main_
